nevergrad/functions/powersystems/test_core.py

```diff
@@ -8,11 +8,14 @@
 from . import core
 
 
-def test_powersystem() -> None:
-    func = core.PowerSystem()
-    x = [7 * np.random.rand(func.dimension // 13) for _ in range(13)]
-    value = func.function(x)  # should not touch boundaries, so value should be < np.inf
-    assert value < np.inf
+def test_powersystem_small() -> None:
+    np.random.seed(12)
+    dams = 2
+    func = core.PowerSystem(num_dams=dams, num_years=0.2)
```

Comment:
using a smaller test case avoids wasting time for nothing, reducing the number of dams and time saves 19s out of 20.